### ISSUE TITLE
Gem is making more calls than necessary

### ISSUE THREAD

**Issue by kurko** [facebook/facebook-ruby-business-sdk#52]
**tl;dr** there are 2 problems, `Batch` doesn't return a response with anything other than ids and the gem makes N requests for N attributes that are called on an object.

## Context

I'm creating many `AdSets` with `FacebookAds::Batch.with_batch`. When I do `batch.execute`, I get back only ids:

```
[
  #<FacebookAds::AdSet {:id=>\"some-id-1\"}>,
  #<FacebookAds::AdSet {:id=>\"some-id-2\"}>,
  #<FacebookAds::AdSet {:id=>\"some-id-3\"}>
]
```

I need to access its `name` and `status` (I want to double check that those were created correctly). When I do `object.first.name`, it makes Request1:

```
Request GET https://graph.facebook.com/v3.0/some-id-1?access_token=my-access-token&appsecret_proof=secret-prood&fields=name
```

Calling `object.first.status`, another request is made:

```
Request GET https://graph.facebook.com/v3.0/some-id-1?access_token=my-access-token&appsecret_proof=secret-prood&fields=status
```

The diff is the `fields` at the end. That led me to just load those ids with `::FacebookAds::AdSet`

## The Problem

I want to load those records, so I do the following for each id:

`::FacebookAds::AdSet.get('some-id-1', 'name,status,campaign_id', session)`

That works, except that in reality I'm creating 3,000 ad sets for N customers. Facebook simply won't accept that many requests in serial. I tried using `FacebookAds::Batch.with_batch` to no avail.

## Questions

1. Is there a way to create records in a batch and get them back with all their attributes, not just `id`?
2. Is there a way to call `object.name` and have it request N attributes at once, instead of one call per attribute? This is very inconvenient.
3. Is there a way to load N adsets by ids without making N requests?

Thanks a lot.

**Comment by kurko** [comment]
Still waiting for some official answer

**Comment by kurko** [comment]
Not fixed.
